src/pkgcheck/checks/codingstyle.py

```diff
@@ -903,3 +903,102 @@ def feed(self, pkg):
                         yield RedundantDodir(
                             cmd.group('cmd'), line=dodir.group('call'),
                             lineno=lineno - 1, pkg=pkg)
+
+
+class EbuildUnquotedVariable(results.LineResult, results.Style):
+    """Variable is unquoted, but should be quoted"""
+
+    def __init__(self, variable, **kwargs):
```

Comment:
I'd change this result to handle multiple line results for a variable to collapse the output of the check. Look at example results such as MissingSlash or similar ones in the codingstyle module.

Reply:
Done.